platforms/dotnet/versions/bookworm/defaultVersion.txt

```diff
@@ -0,0 +1 @@
+6.0
```

Comment:
Should this one be 8?

Reply:
We'd like to default to LTS, which is `6.0`. `8` is not LTS

Reply:
Since .NET8 will eventually become LTS we can switch to .NET8.